docs/getting-started/devenv/using-visual-studio-code.md

```diff
@@ -12,19 +12,387 @@ ms.devlang: dotnet
+file to include the portable profiles in the frameworks section:
+
+```js
+"frameworks": {
+    "netcoreapp1.0": {
+        "imports": [
+            "dnxcore50",
+            "portable-net45+win8"
+        ]
+    }
+}
+```
+
+Finally, you need to add a `testrunner` node to `project.json` that
+tells the .NET SDK that you are using the xUnit test runner. Place this
+node between the `version` and the `dependencies` nodes in your `project.json`
+file.
+
+```js
+"testRunner": "xunit",
+```
+
+Now that the dependencies have been properly configured, let's create
+the tests for your library. Rename `Program.cs` to `LibraryTest.cs` and
+replace its contents with the following code:
+
+```csharp
+using Library;
+using Xunit;
+
+namespace TestApp
+{
+    public class LibraryTests
+    {
+        [Fact]
+        public void TestThing() {
+            Assert.Equal(42, new Thing().Get(19.23));
+        }
+    }
+}
+```
+
+Now, run `dotnet restore`, `dotnet build` and `dotnet test`.
+The xUnit console test runner will run the one test, and report
+that it is passing. 
+
+### Writing the console app
+
+In your terminal, cd to the `golden/src/app` directory. Run `dotnet new`
+to create a new console application.
+
+Your console application depends on the library you built and tested
+in the previous steps. You need to indicate that by editing `project.json`
+to add this dependency.  In the `dependencies` node, add the `Library`
+node below the `Microsoft.NetCore.App` node as follows:
+
+```js
+"dependencies": {
+    "Microsoft.NetCore.App": {
+        "type": "platform",
+        "version": "1.0.0-rc2-3002702"
+    },
+    "library": {
+        "target": "project",
+        "version": "1.0.0-*"
+    }
+}
+```
+
+The `project` node is important here, as it was in the test library. It
+indicates that this is a project in the current solution, and not a
+NuGet package.
+
+Run `dotnet restore` to restore all dependencies. Open `program.cs`
+and replace the contents of the `Main` method with this line:
+
+```cs
+WriteLine($"The answer is {new Thing().Get(19, 23))}");
+```
+
+You'll need to add a couple using directives to the top of the file:
+
+```cs
+using static System.Console;
+using Library;
+```
+
+Then, run `dotnet build`. That creates the assemblies, and you
+can type `dotnet run` to run the executable.
+
+### Debugging your application
+
+You can debug your code in VS Code using the C# extension.
+You install this extension by pressing `F1` to open the VS Code
+palette. Type `ext install` to see the list of extensions. Select the `C#`
+extension. (More details are availble on the [Visual Studio
+Code C# Extension documentation](https://github.com/OmniSharp/omnisharp-vscode/blob/master/debugger.md)
+page.)
+
+After you install the extension, VS Code will ask that you restart the application
+to include the new extension. When you relaunch, VS Code will create a `.vscode` directory
```

Comment:
When I did this, VS Code didn't create the .vscode directory when I relaunched after restarting.  It didn't create .vscode until I tried to start the debugger, then gave me an error message saying I needed to set up my environment.